app/mailers/logbook_entry_mailer.rb

```diff
@@ -0,0 +1,28 @@
+class LogbookEntryMailer < ApplicationMailer
+    before_action :set_user
+    before_action :set_logbook_entry
+  
+    # track_clicks campaign: "logbook_entry_created_email", only: %i[created_email]
+  
+    # has_history extra: -> { {sent: true, messageable: @aircraft} }
+  
+    def created_email 
+      @pilot_in_command = User.find_by(id: @logbook_entry.pilot_in_command_id).first_name
```

Comment:
```suggestion
      @pilot_in_command = @logbook_entry.pilot_in_command
```

Then in the user model you can define a `to_s` method. No change would need to happen to your email subject.
```ruby
# user.rb

def to_s
  "#{first_name} #{last_name}"
end
```